content/blog/2024-06-17-gsoc-multiboot2.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: "Multiboot2 Support in Unikraft"
+description: |
+  Multiboot2, an enhanced version of Multiboot, solves some of the limitations of the original protocol, such as the lack of support for 64-bit UEFI systems. It also provides a more accurate and detailed communication during the bootstrapping process and offers a standardized tag system for easier configuration.
+publishedDate: 2024-06-17
+tags:
+- GSoC'24
+- Multiboot2
+- Booting
+authors:
+- Maria Pana
+---
+
+<img width="100px" src="https://summerofcode.withgoogle.com/assets/media/gsoc-generic-badge.svg" align="right" />
+
+## GSoC'24: Multiboot2 Support in Unikraft
+
+### Project overview
+
+When booting an x86 system, the CPU starts in 16-bit mode and the memory is limited to 1MB (in Real Mode).
+The CPU starts executing at a specific memory address known as a reset vector situated at 0xFFFF0 (for Intel 8086 processors) or 0xFFFFFFF0 (for Intel 80386 and later).
+The initial instructions found here are part of the BIOS, stored in flash memory on the motherboard.
+The BIOS's first goal is to run self test and initialization routines on hardware.
+During this phase, the BIOS sets up standardized interfaces (e.g. legacy BIOS interrupts or UEFI routines) that software that succeeds it (bootloader or operating system) will use to understand the computer's resources without needing special drivers for each different motherboard model.
+
+However, having the kernel directly deal with the interfaces installed by the BIOS unnecessarily increases complexity and maintenance effort.
+This is where the bootloader comes in.
+After its initial routines, BIOS looks for bootable devices, such as a CD-ROM.
```

Comment:
> CD-ROM

Let's just say "such as a Hard Disk". There are several formats for CD-ROMs but I am not sure MBRs/GPTs are used for them and since this is what we give as an example on the next line, let's just go with "Hard Disk" or just "Disk" or SSD, whichever you want.